src/components/views/elements/PersistentApp.js

```diff
@@ -53,16 +57,28 @@ export default class PersistentApp extends React.Component {
         });
     };
 
+    _onMyMembership = async (room) => {
+        const persistentWidgetInRoomId = ActiveWidgetStore.getRoomId(this.state.persistentWidgetId);
+        if (room.getMyMembership() !== "join") {
```

Comment:
```suggestion
    _onMyMembership = async (room, membership) => {
        const persistentWidgetInRoomId = ActiveWidgetStore.getRoomId(this.state.persistentWidgetId);
        if (membership !== "join") {
```

Reply:
well fine then, I'll do the sensible thing :p